src/components/atoms/statusDot/StatusDot.tsx

```diff
@@ -0,0 +1,27 @@
+import { Text } from "@ledgerhq/react-ui";
+import { ReactNode } from "react";
+import "./styles.css";
```

Comment:
I don't really mind going with css here but I think we mostly use styled-components everywhere else